application/src/app/login/services/login.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+import {Observable} from 'rxjs';
+import {Credential} from '../models/login-models';
+
+@Injectable({
+	providedIn: 'root'
+})
+export class LoginService {
+	baseUrl = 'http://localhost:8085';
+
```

Comment:
the login should call Auth server, the url is : localhost:8086/auth/oauth/token?username=tt&password=123&grant_type=password&client_id=oneVoteClient&client_secret=oneVoteClient123

username, password should come from the form.